Admin-app/src/components/SignIn/index.js

```diff
@@ -75,38 +75,43 @@ export default function SignIn() {
   const [email, setEmail] = useState("");
```

Comment:
Why are we touching the "SignIn" page for this feature?  Put in by accident?

Seems to be a fix on the bugs @tusharraval102 is working on.  You should probably revert this.

Reply:
I fixed this to the way Tushar was approaching, but I'll check out his branch and discuss further with you on the server